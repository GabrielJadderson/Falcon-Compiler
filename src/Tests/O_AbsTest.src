# Code generation for the absolute value operator
var a : int;
var b : int;
var c : int;

a = 0 - 7;
b = 6 - 8;
c = 6;

write a;
write b;
write |a|;
write |b|;
write |a| + |c|;
write |a+c|;
