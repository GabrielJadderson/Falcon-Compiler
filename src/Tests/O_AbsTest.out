-7
-2
7
2
13
1
